datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -141,6 +141,12 @@ def start(ctx, check, env, agent, python, dev, base, env_vars, profile_memory):
         env_vars.setdefault(key, value)
 
     if profile_memory:
+        plat = platform.system()
+        try:
+            branch = get_current_branch()
+        except Exception:
+            branch = 'unknown'
```

Comment:
1. Why would that fail ? 🤔 

2. In case it can fail for some reasons, could you log the exception ?

Reply:
1. Maybe repo/check was pulled not via git but rather a tarball
2. Sure!

Reply:
1. But in detached HEAD state. I'm getting this:

```
$ git rev-parse --abbrev-ref HEAD
HEAD
```

The command does not fail.